.github/workflows/check.yml

```diff
@@ -28,14 +52,14 @@ jobs:
           body: |
             Universal binary for Flipper version: ${{ env.flipper_version }}
         env:
-          GITHUB_TOKEN: ${{ github.token }}
+          GITHUB_TOKEN: ${{ secrets.TOKEN }}
```

Comment:
🚨  I was not able to run this workflow on `schedule` without a github token in the SECRETS